test/models/programlocation.js

```diff
@@ -9,7 +9,7 @@ const models = require('../../models');
 
 describe('models.programlocation', () => {
   beforeEach(async () => {
-    await helper.loadFixtures(['programlocation']);
+    await helper.loadFixtures(['program', 'location', 'programlocation']);
```

Comment:
Before we can load fake programlocation fixture data, we need to FIRST load the program and location data, because the new FOREIGN KEY constraints require that the id values actually exists in the corresponding tables first before the rows can be created.